tests/conftest.py

```diff
@@ -1175,3 +1175,20 @@ def mock_revoke_all_tokens(mocked_responses, api_url, account_id, app_id):
         status=200,
         body=json.dumps({"success": True}),
     )
+
+
+@pytest.fixture
+def mock_ip_addresses(mocked_responses, api_url, app_id):
+    ip_addresses_url = "{base}/a/{app_id}/ip_addresses".format(
+        base=api_url, app_id=app_id
+    )
+    mocked_responses.add(
+        responses.GET,
+        ip_addresses_url,
+        content_type="application/json",
+        status=200,
+        body = json.dumps({
+                        "ip_addresses": ["lots-of-ips"],
```

Comment:
Please make the content of this mock actually correct, at least in form. Does this endpoint return IPv4 addresses? If so, I'd suggest using values like `["1.2.3.4", "192.168.0.0", "255.12.147.89"]`, etc. Does this endpoint return IPv6 addresses? If so, I'd suggest using values like `["2001:0db8:0000:0042:0000:8a2e:0370:7334", "2001:0db8:ac10:fe01::", "AbCd:ABCD:abcd::", "2001:0db8:0000:0000:0000:ff00:0042:8329", "::1"]`. Does this endpoint return both? Then mix them up in the mock content.

Reply:
IPv4! I'll make that change.

Reply:
As an aside, does Nylas have any plans to support IPv6 for this feature? Is it possible that this endpoint could return IPv6 values in the future? If so, maybe we should plan for that somehow...

Reply:
To my knowledge no, but I'll check on that. For now I think having IPv4 in the fixture should be sufficient for this PR